src/widgets/Footer/components/FollowUs.tsx

```diff
@@ -0,0 +1,25 @@
+import Icon from '../../../shared/components/Icon';
+
+// interface Props {
+//     colors?: string;
+// }
+
+const FollowUs = () => {
+  // const colorIcon = { 'colors.primary': '10' };
+  return (
+    <div className="flex flex-col mt-12">
+      <p className="font-azeret text-center font-azaret font-medium text-text-secondary">
+        FOLLOW US
+      </p>
+      <div className="flex flex-row justify-center items-center text-center gap-9 mt-12">
+        {/* <Icon icon="twitter" color={`${colorIcon}`} size={'w-[18px]'} /> */}
```

Comment:
I see two similar rows where you want to insert an icon, but this one is commented. Try to get rid of comments.

Reply:
Yes, it will be better to close. I'll try nothing to be lost